commands/utils.go

```diff
@@ -121,9 +124,19 @@ func expandDashRange(s string) ([]int, error) {
 }
 
 type Result struct {
-	Node   int
-	Output testbedi.Output
-	Error  error
+	Node    int
+	Output  testbedi.Output
+	Error   error
+	Elapsed float64
+}
+
+type output struct {
+	Node         int
+	ExitCode     int
+	Errors       error
+	PluginStdOut string
+	PluginStdErr string
```

Comment:
`Errors` => `Error` as it's a single error.

nit: To keep the casing similar to other usages through the code and go in general I think we should use `Stdout` and `Stderr` here.